source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -242,11 +243,26 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         return content;
     }
 
-    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url)
+    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url, winrt::TileControl const& tileControl)
     {
         try
         {
             auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+            auto imagePath = HStringToUTF8(imageUrl.Path());
+            bool isImageSvg = imagePath.find("svg") != std::string::npos;
+
+            if (isImageSvg)
+            {
+                winrt::SvgImageSource svgImageSource{};
+
+                ConfigureSvgImageSourceAsync(renderContext, url, imageUrl, svgImageSource, tileControl);
+                svgImageSource.UriSource(imageUrl);
```

Comment:
This part looks weird. You need to use UriSourceAsync with stream as in https://github.com/microsoft/AdaptiveCards/blob/47f32129cc53547fa8c2fa2574e72decfc12ce08/source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp#L461. You could set this in the ConfigureSvgImageSourceAsync, and skip this line if the SchemaName is "data".